### ISSUE TITLE
Quantitative results of TNT dataset

### ISSUE THREAD

**Issue by 05063112lcs** [TQTQliu/ET-MVSNet#23]
Hello author, this is the result of my reproduction, it turns out to be my problem, but these final indicators are still a little bit behind your paper, what is the reason for this, is there a way to improve it
![image](https://github.com/TQTQliu/ET-MVSNet/assets/149451618/6368e55e-cadd-47fa-855c-3a0d01ad1578)

**Comment by TQTQliu** [comment]
Hello, it is strange that the metric you get is still not good enough, the inermediate part should be around 65.5, advanced part should be around 40.5. I will recently reuse the pre-training weights I provided to run the results and upload the corresponding point cloud.

I'm working on a project right now, so it may take a few days to give you feedback.

**Comment by 05063112lcs** [comment]
> Hello, it is strange that the metric you get is still not good enough, the inermediate part should be around 65.5, advanced part should be around 40.5. I will recently reuse the pre-training weights I provided to run the results and upload the corresponding point cloud.

I tested it with my own model, so I guess it's the effect of the virtual environment。

**Comment by TQTQliu** [comment]
Using our provided pre-trained model,  the inermediate part should be around 65.5 and advanced part should be around 40.5. If the results are produced by your own model, it may be that the trained model is not good enough, or it may be an environmental problem.

**Comment by 05063112lcs** [comment]
> Hello, it is strange that the metric you get is still not good enough, the inermediate part should be around 65.5, advanced part should be around 40.5. I will recently reuse the pre-training weights I provided to run the results and upload the corresponding point cloud.

I tested it with my own model, so I guess it's the effect of the virtual environment。

**Comment by 05063112lcs** [comment]
> Using our provided pre-trained model, the inermediate part should be around 65.5 and advanced part should be around 40.5. If the results are produced by your own model, it may be that the trained model is not good enough, or it may be an environmental problem.
If it's because the training model isn't good enough, how can I make the results close to yours, I haven't made any changes to the code

**Comment by TQTQliu** [comment]
First trained on the DTU dataset, then fine-tuned on the BlendedMVS dataset, we choose the model that has trained after 13 or 14 epoches as the final model to be tested.

**Comment by TQTQliu** [comment]
In addition, for `Horse` scene, you can use DTU-trained models for testing [here](https://github.com/TQTQliu/ET-MVSNet/blob/e29d0b6c1d7772a86b730f4b8b106df3846a7fb2/scripts/test_tnt_inter.sh#L5), and for other scenes you can use fine-tuned models on blendedmvs.

**Comment by 05063112lcs** [comment]
> In addition, for `Horse` scene, you can use DTU-trained models for testing [here](https://github.com/TQTQliu/ET-MVSNet/blob/e29d0b6c1d7772a86b730f4b8b106df3846a7fb2/scripts/test_tnt_inter.sh#L5), and for other scenes you can use fine-tuned models on blendedmvs.

How many rounds of the model to train as the final model, is this determined by the final training loss, I see that your source code says that it is 15 rounds of training, so can I train a few more rounds to get better results
